src/swupd_progress.c

```diff
@@ -0,0 +1,50 @@
+/*
+ *   Software Updater - client side
+ *
+ *      Copyright © 2012-2019 Intel Corporation.
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation, version 2 or later of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "swupd_progress.h"
+#include "swupd.h"
+
+int swupd_progress_callback(void UNUSED_PARAM(*clientp), curl_off_t dltotal, curl_off_t dlnow, curl_off_t UNUSED_PARAM ultotal, curl_off_t UNUSED_PARAM ulnow)
+{
+	struct download_progress *download_progress;
+	double increment;
+
+	download_progress = clientp;
```

Comment:
You are using an parameter set as UNUSED

Reply:
ups, good catch. At first I wasn't using that parameter and I forgot to remove the UNUSED.